.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: csharp
```

Comment:
This file will allow us to configure travis-ci.com to build automatically and even create nuget packages if the build passes